quadrat/sass/templates/_meta.scss

```diff
@@ -2,7 +2,7 @@
 	align-items: center;
 	justify-content: center;
 	@include break-small-only() {
-		padding-top: 0 !important;
+		padding-top: 18px !important;
```

Comment:
I've changed this to `18px` to match the `margin` on the post template container.